React/Base/RCTUtils.m

```diff
@@ -706,15 +706,13 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
     image = [UIImage imageNamed:imageName];
   }
 
-  if (!image) {
+  if (!image && imageURL.isFileURL) {
     // Attempt to load from the file system
-    NSData *fileData;
-    if (imageURL.pathExtension.length == 0) {
-      fileData = [NSData dataWithContentsOfURL:[imageURL URLByAppendingPathExtension:@"png"]];
-    } else {
-      fileData = [NSData dataWithContentsOfURL:imageURL];
+    NSString *filePath = imageURL.path;
```

Comment:
Can you use this instead:

`NSString *filePath = [NSString stringWithUTF8String:[imageURL fileSystemRepresentation]];`